src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -1298,7 +1299,7 @@ private function pullFromAllRemotesUntilFound($hash) {
       $remotes = array_merge($remotes, array_filter(explode(PHP_EOL, $stdin)));
     }
     foreach ($remotes as $remote) {
-      $repository_api->execManualLocal('fetch %s %s', $remote, $hash);
+      $repository_api->execManualLocal('fetch --no-tags --depth=1 %s %s', $remote, $hash);
```

Comment:
As with PR231 - do you have benchmarks showing that it is actually faster?

Reply:
I've removed --no-tags as it does not appear to impact performance anyway. Benchmarks are on PR231.